wiki/Gameplay/Game_modifier/Mod_multiplier/de.md

```diff
@@ -0,0 +1,26 @@
+---
+stub: true
+tags:
+  - score multiplier
+  - score
+  - mod
+  - mod multiplier
+  - Score-Multiplikator
+  - Schwierigkeitsmultiplikator
+---
+
+# Mod-Multiplikator
+
+*Nicht zu verwechseln mit: [Combo](/wiki/Gameplay/Combo_(score_multiplier))*\
+*Siehe auch: [Spielmodifikationen](/wiki/Gameplay/Game_modifier)*
+
+## Beschreibung
+
+Der **Mod-Multiplikator** ist einer der Multiplikatoren, die den [Punktewert](/wiki/Gameplay/Score) eines einzelnen [Hit-Objekts](/wiki/Gameplay/Hit_object) während des Spiels beeinflussen.
+
+Standardmäßig beträgt der Mod-Multiplikator 1,00x. Wenn mehrere [Mods](/wiki/Gameplay/Game_modifier) aktiviert sind, bildet sich der Endwert aus dem Produkt der jeweiligen Mod-Multiplikatoren. Beispielsweise ist der finale Multiplikator in osu! 0,318x, wenn [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock) und [Half Time](/wiki/Gameplay/Game_modifier/Half_Time) gleichzeitig aktiviert sind.
+
+## Trivia
+
+- Der Mod-Multiplikator wird in der [Mod-Auswahl](/wiki/Gameplay/Game_modifier) von osu!(stable) als `Score-Multiplikator` angezeigt, während er in [osu!(lazer)](/wiki/Client/Release_stream/Lazer)  `Schwierigkeitsmultiplikator` heißt.
```

Comment:
For your information, `Score-Multiplikator` is currently used in the game modifier wiki pages (similar to English), while `Schwierigkeitsmultiplikator` matches what is in Crowdin.